scripts/globals/job_utils/thief.lua

```diff
@@ -299,12 +299,18 @@ xi.job_utils.thief.useLarceny = function(player, target, ability, action)
         local newID       = effectStolen:getEffectType()
         local newIcon     = effectStolen:getIcon()
         local newPower    = effectStolen:getPower()
+        local newSubPower = effectStolen:getSubPower()
         local newTick     = effectStolen:getTick()
         local newDuration = effectStolen:getDuration() + jpValue
 
         player:addStatusEffectEx(newID, newIcon, newPower, newTick, newDuration)
         target:delStatusEffect(newID)
 
+        if newSubPower ~= 0 and player:hasStatusEffect(newID) then
+            local newEffect = player:getStatusEffect(newID)
+            newEffect:setSubPower(newSubPower)
+        end
+
```

Comment:
Why not add the subpower directly to the `player:addStatusEffectEx()` above?
Like so
`player:addStatusEffectEx(newID, newIcon, newPower, newTick, newDuration, 0, newSubPower)`

Reply:
Great suggestion! I didn't realize that was an optional parameter.